FirelandsProject/firelands-cata
# (Core/SQL) Script to assemble SQL files

### Changes proposed

- Create a script that allows to generate 1 SQL file for each of the existing databases, so that it is easy to make the importance manually, for those who wish to do so.
- When the pull request containing the modified SQL file was approved, they forgot to add a line at the end of the document, which causes an error when trying to assemble the SQL files.
- Ignore the env folder, so that it is only generated in case the script is used.

### Issues addressed


- Closes

### SOURCE:



### Tests performed


- Windows 10
- Other repositories

### How to Test the Changes:


1. After the pull request is approved, run the script by console, and you will see that inside the env folder, a file is generated because each of the current databases, both for the base and for the updates.

### Known issues and TODO list



---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
